tests/tests.rs

```diff
@@ -200,6 +200,31 @@ fn build_entity() {
     assert_eq!(*world.get::<i32>(f).unwrap(), 789);
 }
 
+#[test]
+fn build_entity_clone() {
+    let mut world = World::new();
+    let mut entity = EntityBuilderClone::new();
+    entity.add("abc");
+    entity.add(123);
+    let e = world.spawn(&entity.build());
+    let mut entity = EntityBuilderClone::new();
+    entity.add("def");
+    entity.add([0u8; 1024]);
+    entity.add(456);
+    entity.add(789);
+    entity.add_bundle(("yup", 67_usize));
+    entity.add_bundle((5.0_f32, String::from("Foo")));
+    entity.add_bundle((7.0_f32, String::from("Bar"), 42_usize));
+    let f = world.spawn(&entity.build());
```

Comment:
Let's spawn multiple times so that any inadvertent memcpying is more likely to trigger a crash.